controller/lib/hal/hal_stm32.h

```diff
@@ -116,14 +116,14 @@ inline SysCtrl_Reg *const SYSCTL_BASE =
     reinterpret_cast<SysCtrl_Reg *>(0xE000E000);
 
 // Interrupt controller
-typedef struct {
+struct IntCtrl_Regs {
```

Comment:
Ya, a lot of this is copied and pasted from a C language header.  I tried to clean all those up, but I guess this one slipped past.